pkg/autodiscovery/autoconfig.go

```diff
@@ -33,6 +33,8 @@ var (
 	errorStats            = newAcErrorStats()
 )
 
+var secretsDecrypt func(data []byte, origin string) ([]byte, error)
```

Comment:
Why not directly assigned it ?

```go
var secretsDecrypt = secrets.Decryt
```

Reply:
You’re definitively right ! => https://github.com/DataDog/datadog-agent/pull/6914/commits/7cfd779f181452c9615c3b192b7d5996abdaef85